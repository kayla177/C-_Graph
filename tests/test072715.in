ENTITY K587VC8A name_K587 paper
ENTITY B7Z9OQVT name_B7Z9 conference
ENTITY NR5N32NX name_NR5N conference
ENTITY J28IA6AT name_J28I conference
ENTITY ENZDBS2M name_ENZD journal
ENTITY L78R9JBB name_L78R paper
ENTITY 7DJYIB28 name_7DJY author
ENTITY Y8U2LSMZ name_Y8U2 institution
ENTITY DMH01CWK name_DMH0 author
ENTITY V41MVMF9 name_V41M institution
RELATIONSHIP 7DJYIB28 cited_by NR5N32NX 1.3
RELATIONSHIP V41MVMF9 presented_at 7DJYIB28 6.6
RELATIONSHIP NR5N32NX authored_by L78R9JBB 1.8
RELATIONSHIP V41MVMF9 published_in L78R9JBB 9.1
RELATIONSHIP 7DJYIB28 published_in L78R9JBB 5.4
RELATIONSHIP K587VC8A affiliated_with DMH01CWK 6.6
RELATIONSHIP V41MVMF9 collaborated_with ENZDBS2M 4.6
RELATIONSHIP 7DJYIB28 collaborated_with Y8U2LSMZ 5.9
RELATIONSHIP DMH01CWK collaborated_with 7DJYIB28 5.7
RELATIONSHIP B7Z9OQVT collaborated_with 7DJYIB28 3.4
RELATIONSHIP ENZDBS2M presented_at NR5N32NX 5.7
RELATIONSHIP B7Z9OQVT affiliated_with K587VC8A 9.0
RELATIONSHIP J28IA6AT cited_by NR5N32NX 0.7
RELATIONSHIP B7Z9OQVT collaborated_with NR5N32NX 2.0
RELATIONSHIP J28IA6AT presented_at B7Z9OQVT 6.7
PATH J28IA6AT Y8U2LSMZ
PRINT ENZDBS2M
PRINT NR5N32NX
PRINT K587VC8A
PATH ENZDBS2M NR5N32NX
DELETE DMH01CWK
HIGHEST
PRINT Y8U2LSMZ
DELETE ENZDBS2M
PATH 7DJYIB28 L78R9JBB
EXIT
