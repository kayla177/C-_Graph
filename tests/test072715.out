success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
J28IA6AT B7Z9OQVT 7DJYIB28 Y8U2LSMZ 16
V41MVMF9 NR5N32NX
7DJYIB28 L78R9JBB ENZDBS2M J28IA6AT B7Z9OQVT
DMH01CWK B7Z9OQVT
ENZDBS2M NR5N32NX 5.7
success
K587VC8A ENZDBS2M 23.6
7DJYIB28
success
7DJYIB28 L78R9JBB 5.4
